### PR TITLE
Fixing Unit Tests

### PR DESCRIPTION
**Pull request by evandana**
## Description

- Fixing functional tests for `wct` in failing branch build on `master` in TravisCI
- IE11 is unreliably passing on 1 test

## TODO

- The branch and local tests were all passing, but the TravisCI tests run on master branch failed. Is there a way to run those before approving a PR?

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "npm-debug.log"
   ],
   "dependencies": {
-    "polymer": "^1.7.0",
+    "polymer": "^1.10.1",
```
Comment (mdwragg) [inline bower.json]: Please revert this, this will force other consumers of the design system to pick this version of Polymer.

File: bower.json [hunk 2]
```diff
@@ -42,7 +42,7 @@
     "px-forms-design": "^2.0.0"
   },
   "resolutions": {
-    "webcomponentsjs": "^0.7.24",
```
Comment (mdwragg) [inline bower.json]: Please revert this also. This is an incorrect mix of webcomponents js and Polymer 1.x

File: polymer.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (mdwragg) [inline polymer.json]: What is this for?

File: test/px-data-table-cell-fixture.html [hunk 4]
```diff
@@ -7,7 +7,7 @@
     <title>px-data-table Demo</title>
 
     <!-- Requires Webcomponents.js polyfill is provided by the page for browsers that don't support html imports -->
-    <script src="../bower_components/webcomponentsjs/webcomponents-lite.min.js"></script>
```
Comment (mdwragg) [inline test/px-data-table-cell-fixture.html]: When you revert webcomponents js, please change this also.

**Comment by evandana** [comment]
Will do those.

For clarity, in the data-cell-fixture file, do you want it to have
bower_components in the import path or not?

**Comment by evandana** [comment]
All tests passing as far as I can tell. Can you make a branch from this or otherwise run the Sauce tests on this PR before it gets merged into Master?

image
image

**Pull request merged**